### ISSUE TITLE
Add Spacy

### ISSUE THREAD

**Issue by julkhami** [rationalmatter/Juno-Issue-Tracker#290]
**What is the name of the package?**
Spacy: https://spacy.io/

I wasn’t able to install this in Juno so I believe it needs to be added.

**Comment by alexstaravoitau** [comment]
Yep, spacy has native extensions — thanks for reporting @julkhami.

**Comment by jaswdr** [comment]
+1

**Comment by swetepete** [comment]
In the package manager it says not supported - which dependencies does it have that can’t run on iOS?

**Comment by alexstaravoitau** [comment]
@swetepete Not sure about dependencies, but SpaCy itself seems to have non-optional native extensions, which need to be compiled for iOS and embedded in the app due to Apple's platform restrictions.
